gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=org.ballerinalang
 version=2.0.0-SNAPSHOT
-ballerinaLangVersion=2201.12.0-20250311-124800-b9003fed
+ballerinaLangVersion=2201.12.0
```

Comment:
Is this change intentional?

Reply:
Yeah. The ballerinalang has been released.

Reply:
Ack

Reply:
they have released u12